masters-golf.md

```diff
@@ -0,0 +1,105 @@
+| Content | Description |
+|---|---|
+| Title               | Swinging into the Future - Golf and Blockchain Technology |
+| Author              | TallAmericano |
+| Featured text       | Is it possible for Golf to be revolutionized by blockchain technology |
+| CT link             | FIXME |
+| Publish date        | 2024-04-21 |
+
+| Social Media | Links |
+|---|---|
+| YouTube link        | FIXME |
+| Podcast link        | FIXME |
+| Bison Relay post    | Title |
+| Reddit link         | FIXME |
+| CT tweet            | FIXME |
+| YouTube tweet       | FIXME |
+| Twitter full video  | FIXME |
+| Twitter clip 1      | FIXME |
+| Twitter clip 2      | FIXME |
+| Matrix post         | Promote your main tweet in "DCR Social Media" channel |
+
+
+## Thesis statement
+
+This article explores the potential integration of blockchain technology into the Golf world
+
+
+## Pre-discussion, research, and questions
+
```

Comment:
To help with the pre-discussion, here you can post a list of questions you're aiming to find the answers to. Make your questions open ended and aim to find at least one contrary points of view to your own. Finding opposite opinions helps us be less biased and leads to more critical thinking.

For example:
We may hate the current handicap grading system but what does it do well? It allows players to keep track of their score and scrutinise officials face to face if they try to increase or decrease it too heavily. In many respects, this is the most purest form of decentralisation.

How will a digital system / blockchain system make this worst? ...

Reply:
I've updated some questions I'd hope to cover in the article